A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -2476,59 +2477,90 @@ class FMCMainDisplay extends BaseAirliners {
     }
 
     setPerfApprQNH(s) {
+        if (s === FMCMainDisplay.clrValue) {
+            this.addNewMessage(NXSystemMessages.notAllowed);
```

Comment:
My theory is that this can be cleared as long as you're the 180 NM out and the PERF APPR page shows cyan brackets. Once you're closer though, and you get the amber boxes, I believe it should be able to clear it anymore. We should probably get pilot confirmation on this and have someone test it.

Reply:
Yes, I did ask for that but not heard anything back yet. I’ll try again. We didn’t support it before so I defaulted to not allowing until I heard something different. I think you are right though, following the general rules, only amber boxes are normally not clearable.